app/build.gradle

```diff
@@ -65,6 +65,10 @@ android {
             }
         }
     }
+
+    lintOptions {
+        disable 'MissingTranslation'
```

Comment:
Let's not do this. This is a slippery slope.

Reply:
The problem is every time anyone creates a string resource they will need to ignore the missing translation.  Is someone with permissions to Lokalise going to go back through and remove these when they update the translations?

Reply:
We can't provide translations for every string in every language? That's what we have the community and Lokalise for. So I agree that we should disable the check for translations, just English should be present.

Reply:
So I suggest that we not disable global lint option. We either use @CedrickFlocon suggestion and disable it internally. However, I prefer that we just add the string right away.

Reply:
I know that for other projects we automatically upload the keys to Lokalise when they are merged to master. Is that something we can set up there too  ?

Reply:
Looks like they have github/git integrations.  https://docs.lokalise.com/en/articles/1684090-github  I don't have lokalise access or repo access to set it up though. 

Even with that setup however the workflow is:
- Merge developers PR
- Lokalise pulls new files from master
- Wait for awesome team to translate
- Lokalise gets updated, PR auto created with new translations.

That still means a dev needs to be able to get a passing build with missing strings.

Reply:
Done. Give me your Lokalise username and I'll add you to the project too.